src/gql/GQLWrapper.tsx

```diff
@@ -40,6 +40,39 @@ interface ClientLinkParams {
 
 const cache = new InMemoryCache({
   typePolicies: {
```

Comment:
This is caused because the keyArgs are omitted. Apollo needs them to determine which keys belong in the cache.  https://www.apollographql.com/docs/react/pagination/key-args#setting-keyargs